static/css/devhub/new-landing/button.less

```diff
@@ -1,13 +1,14 @@
 .Button {
+  background-color: @color-button-default;
   border-radius: 2px;
```

Comment:
I'm making it so the primary button is the modified colour and the default button is the more typical button then a regular button is always the more common style.